transformation/src/main/scala/org/broadinstitute/monster/encode/transformation/EncodeTransformationPipelineBuilder.scala

```diff
@@ -103,4 +105,33 @@ object EncodeTransformationPipelineBuilder extends PipelineBuilder[Args] {
       submittedBy = donorInput.read[String]("submitted_by")
     )
   }
+
+  def transformAntibody(antibodyInput: Msg): Antibody = {
+    // Use regular expressions to remove everything but the actual target name, which should be the same across
+    // all targets in the list. Remove "synthetic_tag" type targets.
+    val mappedTarget = antibodyInput
+      .read[Array[String]]("targets")
+      .collectFirst { case EncodeIdPattern(accession) => accession } // filter out non-matches
+      .flatMap { accession =>
+        val (front, back) = accession.splitAt(accession.lastIndexOf('-'))
+        if (back == "-synthetic_tag") None
+        else Some(front)
+      }
+
+    Antibody(
+      id = antibodyInput.read[String]("accession"),
```

Comment:
I would use `CommonTransformations.readId` here. There were a couple of files with no accession, and I'm wary that it might not be restricted to that object type.